buildSrc/src/main/kotlin/org/jetbrains/reflekt/buildutils/maven-publish-convention.gradle.kts

```diff
@@ -0,0 +1,32 @@
+package org.jetbrains.reflekt.buildutils
+
+import org.gradle.kotlin.dsl.*
+
+plugins {
+    base
+    `maven-publish`
+}
+
+// TODO change
+// JB_SPACE_CLIENT_ID -> ORG_GRADLE_PROJECT_SpacePackagesUsername
+// JB_SPACE_CLIENT_SECRET -> ORG_GRADLE_PROJECT_SpacePackagesPassword
+// and remove 'credentials', gradle will auto-configure based on the maven repo name
+// see https://docs.gradle.org/current/samples/sample_publishing_credentials.html
```

Comment:
Do you think using the Gradle auto-credentials would be useful? If yes, then you'll need to update the environments to use these new variables. If not, then I'll remove this TODO.